docs/settings.rst

```diff
@@ -1000,8 +1000,8 @@ the ``TAG_FEED_ATOM`` and ``TAG_FEED_RSS`` settings:
 
 .. data:: FEED_MAX_ITEMS
```

Comment:
Default should be here with `= 100`.

See `RSS_FEED_SUMMARY_ONLY` below for an example.